Update flags.md (#5103)
FerretDB/FerretDB: website/docs/configuration/flags.md

To make user experience cloud native, every flag has its environment variable equivalent.
There is no configuration file.

However, you can use a bash script to configure and run FerretDB. Here's an example:

```bash
#!/bin/sh
export FERRETDB_POSTGRESQL_URL=xxx

> exec ferretdb

Reviewer: Could you please document the necessity of `exec` for handling termination signals.

Author: Ok

To make user experience cloud native, every flag has its environment variable equivalent.
There is no configuration file.

However, you can use a bash script to configure and run FerretDB. Here's an example:

> ```bash

Reviewer 2: ```suggestion
```sh
```

#!/bin/sh

> export FERRETDB_POSTGRESQL_URL=xxx

Reviewer: Could you specify more realistic URL instead of xxx please?

exec ferretdb
```
This approach avoids long command lines and improves clarity.

> The use of `exec` allows the FerretDB process to correctly receive and handle termination signals (like SIGTERM). This is particularly important when running FerretDB as PID 1 in containers, ensuring a clean and graceful shutdown.

Reviewer 3: While the part about containers is correct, why would one add a script to the container already configured by environment variables?

A more realistic example would be someone starting the FerretDB binary manually on the host, ensuring that command-line flags are not lost in the shell history and that the environment is not reset in a new terminal window.

> Note: The use of exec is important—it replaces the shell with the FerretDB process, allowing it to correctly receive and handle termination signals (like SIGTERM). This is particularly important when running FerretDB as PID 1 in containers, ensuring a clean and graceful shutdown.

Reviewer: ```suggestion
The use of `exec` allows the FerretDB process to correctly receive and handle termination signals (like SIGTERM). This is particularly important when running FerretDB as PID 1 in containers, ensuring a clean and graceful shutdown.
```
